W-22395996 added other supported assets (#91)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/viewing-api-details.adoc

> = Viewing Service Details

Reviewer: ```suggestion
= Viewing API Details
```

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> To view the Service Details page, select a service from the Home or Services page of the portal. 

Reviewer: ```suggestion
To view the API Details page, select a service from the Home or APIs page of the portal. 
```

// add screen captures

> The Service Details page includes the following information about a service:

Reviewer: ```suggestion
The API Details page includes the following information about a service:
```

* *Name*
+
Name of the service.
* *Type*
+ 

> The service type can be agents, APIs, LLMs, and MCP servers. AsyncAPI, GraphQL, gRPC, HTML, REST, and SOAP APIs are supported.

Reviewer: @JennyHajee This sentence feels off. Provided a suggestion below to try and unify the two sentences but feel free to change as needed. 
```suggestion
The service type can be Agents, APIs, LLMs, MCP servers. AsyncAPI, GraphQL, gRPC, HTML, REST, and SOAP APIs are also supported.
```

* *Contact email*
+
The email of the contact for the service.

* *Categories*
+
The name and value of the predefined category this service belongs to. Services can have multiple categories. Categories organize services into groups to improve browsing and discovery. Categories are created and maintained in Exchange.
* *Tags*
+
The name of a predefined keyword associated with this service. Services can have multiple tags. Tags are created and maintained in Exchange. 
* *Description*
+
A description of the service.

* *Asset version*
+
The version of the service.
* *Version state*
+
The lifecycle state (development, stable, or deprecated) of this service. 
* *Instances*
+
The count of instances of the service.
* *Clients*
+
The number of client applications of the service.

The API Console is located in the Services Details page. The console component is interactive and shows information about API specifications and their methods. Endpoints are displayed in alphabetical order. See xref:using-the-api-console.adoc[] for more information.

== View Categories and Tags
To view the categories or tags for a service:

> . On the *Service details* page, click *Categories* or *Tags* to view the information.

Reviewer: ```suggestion
. On the *API details* page, click *Categories* or *Tags* to view the information.
```

. Click *Categories* or *Tags* again to close the list.

== View Instances Information and Copy the URL
To view the name and URL of an instance and copy the URL:

> . On the *Service details* page, click *View instances*.

Reviewer: ```suggestion
. On the *API details* page, click *View instances*.
```
